apps/protailwind/src/video/video.tsx

```diff
@@ -82,7 +82,7 @@ export const Video: React.FC<
             hidden: displayOverlay,
           })}
         >
-          {canShowVideo && !loadingVideoResource ? (
+          {canShowVideo && videoResource?.muxPlaybackId ? (
```

Comment:
we want to make sure the data is available and do not care about the loading status of the resource

trpc (react-query) is `loading` on focus, but the video `muxPlaybackId` should be stable and present when reloading.